iree/compiler/Codegen/Common/ConvertToDestinationPassingStylePass.cpp

```diff
@@ -304,10 +310,84 @@ static LogicalResult duplicateInitTensorOps(OpBuilder &b,
   return success();
 }
 
+namespace {
+SmallVector<NamedAttribute> PruneAttributeList(linalg::GenericOp op) {
```

Comment:
Why do we need this?

Reply:
This was needed because the op could carry other attributes. E.g., lowering_config.